akka-docs/rst/java/stream/stages-overview.rst

```diff
@@ -1160,6 +1160,20 @@ If materialized values needs to be collected ``prependMat`` is available.
 
 **completes** when all upstreams complete
 
+orElse
+^^^^^^
+If the first source completes without emitting any elements, the elements from the second source
+are emitted. If the first source emits any elements the second source is cancelled.
+
+Signal errors downstream, regardless which of the two sources emitted the error.
+
+**emits** when an element is available from first stream or first stream closed without emitting any elements and an element
+is available from the second stream
+
+**backpressures** when downstream backpressures
+
+**completes** when the first stream completes after emitting at least one element or else when the second stream completes
```

Comment:
I think this definition is missing one case. What happens if the second stream completes before the first? I assume that it will then complete when the first stream completes